store/sqlstore/webhook_store.go

```diff
@@ -331,26 +331,25 @@ func (s SqlWebhookStore) UpdateOutgoing(hook *model.OutgoingWebhook) store.Store
 	})
 }
 
-func (s SqlWebhookStore) AnalyticsIncomingCount(teamId string) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		query :=
-			`SELECT 
+func (s SqlWebhookStore) AnalyticsIncomingCount(teamId string) (int64, *model.AppError) {
+	query :=
+		`SELECT 
 			    COUNT(*)
```

Comment:
nit: I would unindent this lines one level, to keep it consistent with the previous implementation.

Reply:
@jespino is it correct now? Changed my editor indentation settings to tabs .